sale_margin_update/models/sale_order_line.py

```diff
@@ -0,0 +1,19 @@
+###############################################################################
```

Comment:
This is not OCA guidelines. You can put here short license headers.

See this example: https://github.com/OCA/maintainer-tools/blob/master/template/module/models/model_name.py#L1